templates/flavors/windows-containerd/kustomization.yaml

```diff
@@ -2,8 +2,16 @@ namespace: default
 resources:
   - ../default
   - machine-deployment-windows.yaml
+  - ../../addons/windows/csi-proxy/csi-proxy-resource-set.yaml
```

Comment:
We have not been installing Cluster Resource Sets for the default templates, only for the E2E tests.  This is mostly because cluster api doesn't include those definitions in what is considered a cluster.

 We shouldn't include this in the default template as all addon and other components (such as cni) needs to be managed by the user after the base cluster is up and running.

Since CAPI doesn't deploy CNI's if a customer used this template with a CRS I think they would get errors since they `clusterctl generate cluster <template>` doesn't include a CNI.  I think CRS are also an experimental feature?

The way we are handling this as of today with tilt is to deploy during tilt initialization like:

https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/e2abea2a0139ff9f8fd5e55dccc31a06208088e2/Tiltfile#L223-L225

Reply:
Thanks, I was able to get things working this way for dev clusters!